backend/test/mastodon/media.spec.ts

```diff
@@ -12,8 +12,11 @@ const domain = 'cloudflare.com'
 describe('Mastodon APIs', () => {
 	describe('media', () => {
 		test('upload image creates object', async () => {
-			globalThis.fetch = async (input: any) => {
-				if (input.url.toString() === 'https://api.cloudflare.com/client/v4/accounts/testaccountid/images/v1') {
+			globalThis.fetch = async (input: unknown) => {
+				if (
+					(input as { url: URL }).url.toString() ===
+					'https://api.cloudflare.com/client/v4/accounts/testaccountid/images/v1'
+				) {
 					return new Response(
```

Comment:
Similar to `follow.spec.ts`, something like:

```ts
			globalThis.fetch = async (input: RequestInfo) => {
				const request = new Request(input)
				if (request.url === 'https://api.cloudflare.com/client/v4/accounts/testaccountid/images/v1') {
```

Reply:
sorry I missed this one 😓 👍